### PR TITLE
Fixes #694 - Fixed vATIS transition levels

### PR DESCRIPTION
**Pull request by lewishh**
Fixes #694

# Summary of changes

TODO

# Screenshots (if necessary)

< Please provide screenshots if this will facilitate a faster review of these changes >

### CODE REVIEW

File: UK/vATIS/ADC/Aberdeen(EGPD).json [hunk 1]
```diff
@@ -118,35 +118,40 @@
...
               "low": 1032,
               "high": 1049,
-              "altitude": 55
+              "altitude": 65
             },
             {
               "low": 1013,
               "high": 1031,
-              "altitude": 60
+              "altitude": 70
```
Comment (lewishh) [inline UK/vATIS/ADC/Aberdeen(EGPD).json:134]: Note: Aberdeen 1013 hPa classified as high-pressure.
#Scottish vMATS 4.2 Note 1.

File: UK/vATIS/ADC/Isle of Man(EGNS).json [hunk 2]
```diff
@@ -127,35 +127,40 @@
...
+              "low": 1014,
               "high": 1031,
-              "altitude": 60
+              "altitude": 40
             },
             {
               "low": 995,
-              "high": 1012,
-              "altitude": 65
+              "high": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Isle of Man(EGNS).json:147]: 1013 hPa considered low-pressure.
#Isle of Man ACB

File: UK/vATIS/ADC/Lydd(EGMD).json [hunk 3]
```diff
@@ -118,19 +118,24 @@
...
+                            "altitude": 60
+                        },
                         {
                             "low": 1032,
                             "high": 1049,
                             "altitude": 65
                         },
                         {
-                            "low": 1014,
+                            "low": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Lydd(EGMD).json:132]: 1013 hPa considered high-pressure.
#Lydd vMATS 1.1.6

File: UK/vATIS/ADC/Northumberland(EGNT & EGNV).json [hunk 4]
```diff
@@ -276,35 +276,40 @@
...
+              "low": 1014,
               "high": 1031,
-              "altitude": 60
+              "altitude": 70
             },
             {
               "low": 995,
-              "high": 1012,
-              "altitude": 65
+              "high": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Northumberland(EGNT & EGNV).json:296]: 1013 hPa considered low-pressure.
# London vMATS APT-58 17.2.1

File: UK/vATIS/ADC/Norwich(EGSH).json [hunk 5]
```diff
@@ -113,19 +113,24 @@
...
             {
-              "low": 1013,
+              "low": 1014,
               "high": 1031,
               "altitude": 60
             },
             {
               "low": 995,
-              "high": 1012,
+              "high": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Norwich(EGSH).json:133]: 1013 hPa considered low-pressure.
#London vMATS APT-61 18.2.1

File: UK/vATIS/ADC/Prestwick(EGPK).json [hunk 6]
```diff
@@ -113,35 +113,40 @@
...
+              "low": 1014,
               "high": 1031,
-              "altitude": 60
+              "altitude": 70
             },
             {
               "low": 995,
-              "high": 1012,
-              "altitude": 65
+              "high": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Prestwick(EGPK).json:133]: 1013 hPa considered low-pressure in Scottish TMA.

File: UK/vATIS/ADC/Shoreham(EGKA).json [hunk 7]
```diff
@@ -134,19 +134,24 @@
...
+                            "altitude": 60
+                        },
                         {
                             "low": 1032,
                             "high": 1049,
                             "altitude": 65
                         },
                         {
-                            "low": 1014,
+                            "low": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Shoreham(EGKA).json:148]: 1013 hPa considered high-pressure.
#Shoreham vMATS

File: UK/vATIS/UK - AC North.json [hunk 8]
```diff
@@ -344,13 +342,13 @@
               "Altitude": 55
             },
             {
-              "Low": 1014,
+              "Low": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - AC North.json:345]: 1013 hPa is considered high-pressure.
#Manchester vMATS 1.5

File: UK/vATIS/UK - AC North.json [hunk 9]
```diff
@@ -1052,13 +1056,13 @@
...
```
Comment (lewishh) [inline UK/vATIS/UK - AC North.json:1059]: 1013 hPa is considered high-pressure.
#Liverpool vMATS 1.5

File: UK/vATIS/UK - AC North.json [hunk 10]
```diff
@@ -1411,13 +1417,13 @@
               "Altitude": 35
             },
             {
-              "Low": 1013,
+              "Low": 1014,
```
Comment (lewishh) [inline UK/vATIS/UK - AC North.json:1420]: 1013 hPa considered low-pressure.
#Isle of Man ACB

File: UK/vATIS/UK - AC North.json [hunk 11]
```diff
@@ -2832,13 +2846,13 @@
...
             {
-              "Low": 1013,
+              "Low": 1014,
               "High": 1031,
               "Altitude": 60
             },
             {
               "Low": 995,
-              "High": 1012,
+              "High": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - AC North.json:2855]: 1013 hPa considered low-pressure.
#London vMATS APT-61 18.2.1

File: UK/vATIS/UK - AC North.json [hunk 12]
```diff
@@ -3542,32 +3560,37 @@
...
+              "Low": 1014,
               "High": 1031,
-              "Altitude": 30
+              "Altitude": 40
             },
             {
               "Low": 995,
-              "High": 1012,
-              "Altitude": 35
+              "High": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - AC North.json:3577]: 1013 hPa considered low pressure. No reference available.

File: UK/vATIS/UK - AC North.json [hunk 13]
```diff
@@ -3901,32 +3921,37 @@
...
+              "Low": 1014,
               "High": 1031,
-              "Altitude": 30
+              "Altitude": 70
             },
             {
               "Low": 995,
-              "High": 1012,
-              "Altitude": 35
+              "High": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - AC North.json:3938]: 1013 hPa considered low-pressure.
#Newcastle vMATS 1.5

File: UK/vATIS/UK - LON_CTR Only.json [hunk 14]
```diff
@@ -2456,13 +2453,13 @@
               "Altitude": 55
             },
             {
-              "Low": 1014,
+              "Low": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - LON_CTR Only.json:2456]: 1013 hPa considered high-pressure
#Manchester vMATS 1.5

File: UK/vATIS/UK - LON_CTR Only.json [hunk 15]
```diff
@@ -3481,13 +3488,13 @@
...
```
Comment (lewishh) [inline UK/vATIS/UK - LON_CTR Only.json:3491]: 1013 hPa considered high pressure.
#Liverpool vMATS 1.5

File: UK/vATIS/ADC/Oxford(EGTK).json [hunk 16]
```diff
@@ -118,19 +118,24 @@
...
+                            "altitude": 60
+                        },
                         {
                             "low": 1032,
                             "high": 1049,
                             "altitude": 65
                         },
                         {
-                            "low": 1014,
+                            "low": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Oxford(EGTK).json:132]: It's unclear if this is true except from ukcp datasource.
#https://ukcp.vatsim.uk/airfields/7

File: UK/vATIS/ADC/Cranfield(EGTC).json [hunk 17]
```diff
@@ -118,19 +118,24 @@
...
+                            "altitude": 60
+                        },
                         {
                             "low": 1032,
                             "high": 1049,
                             "altitude": 65
                         },
                         {
-                            "low": 1014,
+                            "low": 1013,
```
Comment (lewishh) [inline UK/vATIS/ADC/Cranfield(EGTC).json:132]: It's unclear if this is true except ukcp datasource.
https://ukcp.vatsim.uk/airfields/53

File: UK/vATIS/UK - PC MAN.json [hunk 18]
```diff
@@ -368,13 +366,13 @@
               "Altitude": 55
             },
             {
-              "Low": 1014,
+              "Low": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - PC MAN.json:369]: 1013 hPa considered high pressure.
#Manchester vMATS 1.5

File: UK/vATIS/UK - PC MAN.json [hunk 19]
```diff
@@ -1112,13 +1116,13 @@
...
```
Comment (lewishh) [inline UK/vATIS/UK - PC MAN.json:1119]: 1013 hPa considered high pressure.
#Liverpool vMATS 1.5

File: UK/vATIS/UK - PC MAN.json [hunk 20]
```diff
@@ -1495,13 +1501,13 @@
               "Altitude": 35
             },
             {
-              "Low": 1013,
+              "Low": 1014,
```
Comment (lewishh) [inline UK/vATIS/UK - PC MAN.json:1504]: 1013 hPa considered low-pressure.
#Isle of Man ACB

File: UK/vATIS/UK - PC MAN.json [hunk 21]
```diff
@@ -2626,32 +2638,37 @@
...
               "High": 1031,
-              "Altitude": 60
+              "Altitude": 40
             },
             {
               "Low": 995,
-              "High": 1012,
-              "Altitude": 65
+              "High": 1013,
```
Comment (lewishh) [inline UK/vATIS/UK - PC MAN.json:2655]: 1013 hPa considered low pressure.
#https://ukcp.vatsim.uk/airfields/41

**Comment by luke11brown** [comment]
Closed as covered in #696 #697 #698 #699 #700 #701 #702 #703 #704 #705 #706 #707 #708 #709 #710 #711 #712 #713 #714 #715 #716 #717 #718 #719

**Pull request closed without merging**